me.hyliu.fluentreader.yml

```diff
@@ -0,0 +1,80 @@
+app-id: me.hyliu.fluentreader
+runtime: org.freedesktop.Platform
+runtime-version: "20.08"
+branch: stable
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "20.08"
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node14
+command: start-fluent-reader
```

Comment:
Does this work? Down there you create the script as `start-fluent-reader.sh`

Reply:
Yes. Moved to `start-fluent-reader`: https://github.com/flathub/flathub/blob/9a64ac64cef380a4fc894528cc634d26a5ab1360/me.hyliu.fluentreader.yml#L50

Kind of "trick" borrowed from https://github.com/flatpak/flatpak-builder-tools/blob/6387cd8645900c926f75768d530047d437addc8f/node/vanilla-quick-start/org.electronjs.ElectronQuickStart.yaml#L60

Reply:
sorry I missed that line. 

all good.